lizard_connector/connector.py

```diff
@@ -0,0 +1,342 @@
+# coding=utf-8
+"""
+Connector to Lizard api (e.g. https://demo.lizard.net/api/v2) for python.
+
+Includes:
+- Endpoints (Lizard api endoints, e.g. TimeseriesEndpoint)
+- Fetcher (http handling)
+- queryfunctions for special cases such as geographical queries and time
+related queries other queries can be input as a dictionary
+- parserfunctions to parse the json obtained from Endpoint queries
+"""
+
+import json
+import urllib.request
+import urllib.parse
+
+from lizard_connector.query import QueryDictionary
+
+
+class LizardApiTooManyResults(Exception):
+    pass
+
+
+class Fetcher(object):
+
+    def __init__(self, endpoint, base="https://demo.lizard.net",
```

Comment:
@arjenvrielink, zie de opmerking van Reinout hierboven, is demo.lizard.net de beste keus?